--- a/Common/kernel.pro	2026-05-30 17:28:43.765603762 +0800
+++ b/Common/kernel.pro	2026-05-29 12:08:48.951297387 +0800
@@ -17,13 +17,15 @@
 PWD_ROOT_DIR = $$PWD
 include(../Common/base.pri)
 
-CONFIG += build_all_zlib build_zlib_as_sources
+# CryptPad: This causes duplicate symbols when linking x2t
+# CONFIG += build_all_zlib build_zlib_as_sources
 include(../OfficeUtils/OfficeUtils.pri)
 
 CONFIG += core_static_link_xml_full
 include(../DesktopEditor/xml/build/qt/libxml2.pri)
 
-ADD_DEPENDENCY(UnicodeConverter)
+# CryptPad: UnicodeConverter is linked in a later step. Do not link here.
+# ADD_DEPENDENCY(UnicodeConverter)
 
 # CONFIG
 HEADERS += ./kernel_config.h
