--- a/DesktopEditor/graphics/pro/raster.pri	2026-05-30 17:33:57.266669144 +0800
+++ b/DesktopEditor/graphics/pro/raster.pri	2026-05-29 12:08:49.487771590 +0800
@@ -357,18 +357,19 @@
 		$$OFFICEUTILS_PATH/src/zlib-1.2.11 \
 		$$OFFICEUTILS_PATH/src
 
-	SOURCES += \
-		$$OFFICEUTILS_PATH/src/zlib-1.2.11/crc32.c \
-		$$OFFICEUTILS_PATH/src/zlib-1.2.11/adler32.c \
-		$$OFFICEUTILS_PATH/src/zlib-1.2.11/deflate.c \
-		$$OFFICEUTILS_PATH/src/zlib-1.2.11/inffast.c \
-		$$OFFICEUTILS_PATH/src/zlib-1.2.11/inflate.c \
-		$$OFFICEUTILS_PATH/src/zlib-1.2.11/inftrees.c \
-		$$OFFICEUTILS_PATH/src/zlib-1.2.11/trees.c \
-		$$OFFICEUTILS_PATH/src/zlib-1.2.11/zutil.c \
-		$$OFFICEUTILS_PATH/src/zlib-1.2.11/uncompr.c \
-		$$OFFICEUTILS_PATH/src/zlib-1.2.11/compress.c \
-		$$OFFICEUTILS_PATH/src/zlib_addon.c
+# CryptPad: Do not include zlib here, it is already included in the common module.
+#	SOURCES += \
+#		$$OFFICEUTILS_PATH/src/zlib-1.2.11/crc32.c \
+#		$$OFFICEUTILS_PATH/src/zlib-1.2.11/adler32.c \
+#		$$OFFICEUTILS_PATH/src/zlib-1.2.11/deflate.c \
+#		$$OFFICEUTILS_PATH/src/zlib-1.2.11/inffast.c \
+#		$$OFFICEUTILS_PATH/src/zlib-1.2.11/inflate.c \
+#		$$OFFICEUTILS_PATH/src/zlib-1.2.11/inftrees.c \
+#		$$OFFICEUTILS_PATH/src/zlib-1.2.11/trees.c \
+#		$$OFFICEUTILS_PATH/src/zlib-1.2.11/zutil.c \
+#		$$OFFICEUTILS_PATH/src/zlib-1.2.11/uncompr.c \
+#		$$OFFICEUTILS_PATH/src/zlib-1.2.11/compress.c \
+#		$$OFFICEUTILS_PATH/src/zlib_addon.c
 
 	CONFIG += config_zlip_image
 }
