--- a/HtmlFile2/htmlfile2.cpp	2026-05-30 17:36:37.369821356 +0800
+++ b/HtmlFile2/htmlfile2.cpp	2026-05-29 12:08:50.279757291 +0800
@@ -252,7 +252,7 @@
 	std::wstring sGridSpan = L"1";
 	std::wstring sPr = L"";
 
-	CTc(int _i, int _j, const std::wstring& sColspan, const std::wstring& sTcPr = L"") 
+	CTc(int _i, int _j, const std::wstring& sColspan, const std::wstring& sTcPr = L"")
 		: i(_i), j(_j), sGridSpan(sColspan), sPr(sTcPr) {}
 
 	bool operator==(const CTc& c2)
@@ -593,7 +593,7 @@
 class CTableCell
 {
 public:
-	CTableCell() 
+	CTableCell()
 		: m_unColspan(1), m_unRowSpan(1), m_bIsMerged(false), m_bIsEmpty(false)
 	{}
 
@@ -602,7 +602,7 @@
 	{}
 
 	CTableCell(CTableCell& oCell)
-		: m_unColspan(oCell.m_unColspan), m_unRowSpan(oCell.m_unRowSpan), m_bIsMerged(oCell.m_bIsMerged), 
+		: m_unColspan(oCell.m_unColspan), m_unRowSpan(oCell.m_unRowSpan), m_bIsMerged(oCell.m_bIsMerged),
 		  m_bIsEmpty(oCell.m_bIsEmpty), m_oStyles(oCell.m_oStyles)
 	{
 		WriteToStringBuilder(oCell.m_oData, m_oData);
@@ -782,12 +782,12 @@
 				--m_oStyles.m_unMaxIndex;
 				delete *itFoundEmpty;
 				*itFoundEmpty = pCell;
-				
+
 				if (1 != pCell->GetColspan())
 				{
 					++itFoundEmpty;
 					UINT unColspan = pCell->GetColspan() - 1;
-	
+
 					while (m_arCells.end() != itFoundEmpty && (*itFoundEmpty)->Empty() && unColspan > 0)
 					{
 						--m_oStyles.m_unMaxIndex;
@@ -795,7 +795,7 @@
 						delete (*itFoundEmpty);
 						itFoundEmpty = m_arCells.erase(itFoundEmpty);
 					}
-	
+
 					if (unColspan != 0)
 						pCell->SetColspan(pCell->GetColspan() - unColspan, MAXCOLUMNSINTABLE);
 				}
@@ -1185,7 +1185,7 @@
 				if (0 != unMaxIndex && 1 == pRow->GetCount() && pRow->GetIndex() > unMaxIndex)
 				{
 					pCell = (*pRow)[unIndex];
-	
+
 					if (NULL == pCell)
 						continue;
 
@@ -1376,7 +1376,7 @@
 	UpperRoman
 };
 
-void replace_all(std::wstring& s, const std::wstring& s1, const std::wstring& s2)
+void htmlfile2replace_all(std::wstring& s, const std::wstring& s1, const std::wstring& s2)
 {
 	size_t pos = s.find(s1);
 	size_t l = s2.length();
@@ -1523,18 +1523,18 @@
 	return false;
 }
 
-std::wstring EncodeXmlString(const std::wstring& s)
+std::wstring HtmlFile2EncodeXmlString(const std::wstring& s)
 {
 	std::wstring sRes = s;
 
-	replace_all(sRes, L"&", L"&amp;");
-	replace_all(sRes, L"<", L"&lt;");
-	replace_all(sRes, L">", L"&gt;");
-	replace_all(sRes, L"\"", L"&quot;");
-	replace_all(sRes, L"\'", L"&#39;");
-	replace_all(sRes, L"\n", L"&#xA;");
-	replace_all(sRes, L"\r", L"&#xD;");
-	replace_all(sRes, L"\t", L"&#x9;");
+	htmlfile2replace_all(sRes, L"&", L"&amp;");
+	htmlfile2replace_all(sRes, L"<", L"&lt;");
+	htmlfile2replace_all(sRes, L">", L"&gt;");
+	htmlfile2replace_all(sRes, L"\"", L"&quot;");
+	htmlfile2replace_all(sRes, L"\'", L"&#39;");
+	htmlfile2replace_all(sRes, L"\n", L"&#xA;");
+	htmlfile2replace_all(sRes, L"\r", L"&#xD;");
+	htmlfile2replace_all(sRes, L"\t", L"&#x9;");
 
 	return sRes;
 }
@@ -1628,7 +1628,7 @@
 	NSFonts::IApplicationFonts*          m_pFonts;     // Необходимо для оптимизации работы со шрифтами
 public:
 
-	CHtmlFile2_Private() 
+	CHtmlFile2_Private()
 		: m_nFootnoteId(1), m_nHyperlinkId(1), m_nNumberingId(1), m_nId(1), m_nShapeId(1), m_pFonts(NULL)
 	{
 		m_oPageData.SetWidth (DEFAULT_PAGE_WIDTH,  NSCSS::UnitMeasure::Twips, 0, true);
@@ -1759,31 +1759,31 @@
 			if(!oParams->m_sBookTitle.empty())
 			{
 				sCore += L"<dc:title>";
-				sCore += EncodeXmlString(oParams->m_sBookTitle);
+				sCore += HtmlFile2EncodeXmlString(oParams->m_sBookTitle);
 				sCore += L"</dc:title>";
 			}
 			if(!oParams->m_sAuthors.empty())
 			{
 				sCore += L"<dc:creator>";
-				sCore += EncodeXmlString(oParams->m_sAuthors);
+				sCore += HtmlFile2EncodeXmlString(oParams->m_sAuthors);
 				sCore += L"</dc:creator>";
 			}
 			if(!oParams->m_sGenres.empty())
 			{
 				sCore += L"<dc:subject>";
-				sCore += EncodeXmlString(oParams->m_sGenres);
+				sCore += HtmlFile2EncodeXmlString(oParams->m_sGenres);
 				sCore += L"</dc:subject>";
 			}
 			if(!oParams->m_sDate.empty())
 			{
 				sCore += L"<dcterms:created xsi:type=\"dcterms:W3CDTF\">";
-				sCore += EncodeXmlString(oParams->m_sDate);
+				sCore += HtmlFile2EncodeXmlString(oParams->m_sDate);
 				sCore += L"</dcterms:created>";
 			}
 			if(!oParams->m_sDescription.empty())
 			{
 				sCore += L"<dc:description>";
-				sCore += EncodeXmlString(oParams->m_sDescription);
+				sCore += HtmlFile2EncodeXmlString(oParams->m_sDescription);
 				sCore += L"</dc:description>";
 			}
 			if (!oParams->m_sLanguage.empty())
@@ -1914,7 +1914,7 @@
 		m_oDocXml.WriteString(L"w:header=\"" + std::to_wstring(m_oPageData.GetHeader().ToInt(NSCSS::Twips)) + L"\" ");
 		m_oDocXml.WriteString(L"w:footer=\"" + std::to_wstring(m_oPageData.GetFooter().ToInt(NSCSS::Twips)) + L"\" ");
 		m_oDocXml.WriteString(L"w:gutter=\"0\"/><w:cols w:space=\"720\"/><w:docGrid w:linePitch=\"360\"/></w:sectPr></w:body></w:document>");
-		
+
 		NSFile::CFileBinary oDocumentWriter;
 		if (oDocumentWriter.CreateFileW(m_sDst + L"/word/document.xml"))
 		{
@@ -2430,7 +2430,7 @@
 		{
 			std::wstring sName  = m_oLightReader.GetName();
 			if(sName == L"class")
-				oNode.m_wsClass  = EncodeXmlString(m_oLightReader.GetText());
+				oNode.m_wsClass  = HtmlFile2EncodeXmlString(m_oLightReader.GetText());
 			else if(sName == L"id")
 			{
 				oNode.m_wsId = EncodeXmlString(m_oLightReader.GetText());
@@ -2485,9 +2485,9 @@
 	void readBody()
 	{
 		std::vector<NSCSS::CNode> sSelectors;
-		
+
 		sSelectors.push_back(NSCSS::CNode(L"html", L"", L""));
-		
+
 		GetSubClass(&m_oDocXml, sSelectors);
 
 		if (!sSelectors.back().m_mAttributes.empty())
@@ -3661,7 +3661,7 @@
 			pColgroup->AddCol(pCol);
 		}
 	}
-	
+
 	struct TRowspanElement
 	{
 		UINT  m_unRowSpan;
@@ -3721,7 +3721,7 @@
 					else if(m_oLightReader.GetName() == L"rowspan")
 					{
 						pCell->SetRowspan(NSStringFinder::ToInt(m_oLightReader.GetText(), 1));
-						
+
 						if (1 != pCell->GetRowspan())
 							arRowspanElements.push_back({pCell->GetRowspan(), unColumnIndex, pCell});
 					}
@@ -4007,7 +4007,7 @@
 		//------
 
 		int nDeath = m_oLightReader.GetDepth();
-		while(m_oLightReader.ReadNextSiblingNode(nDeath)) 
+		while(m_oLightReader.ReadNextSiblingNode(nDeath))
 		{
 			const std::wstring sName = m_oLightReader.GetName();
 			GetSubClass(oXml, sSelectors);
