--- a/OdfFile/Projects/Linux/OdfFormatLib.pro	2026-05-30 17:36:37.802135534 +0800
+++ b/OdfFile/Projects/Linux/OdfFormatLib.pro	2026-05-29 12:08:51.254836460 +0800
@@ -41,7 +41,7 @@
 }
 
 core_debug {
-SOURCES += \    
+SOURCES += \
         ../../DataTypes/referenceformat.cpp \
         ../../DataTypes/mathvariant.cpp \
 	../../DataTypes/anchortype.cpp \
@@ -638,7 +638,7 @@
 	../../Reader/Format/math_table_elements.h \
 	../../Reader/Format/math_token_elements.h \
 	../../Reader/Format/calcext_elements.h \
-	../../Reader/Format/table_database_ranges.h \		
+	../../Reader/Format/table_database_ranges.h \
 	../../Reader/Converter/docx_content_type.h \
 	../../Reader/Converter/docx_conversion_context.h \
 	../../Reader/Converter/docx_conversion_state.h \
@@ -830,3 +830,135 @@
 	../../Writer/Converter/XlsxConverter.h \
 	../../Writer/Converter/PptxConverter.h \
 
+HEADERS += \
+        ../../../OOXML/Base/Base.h \
+        ../../../OOXML/Base/Nullable.h \
+        ../../../OOXML/Base/SmartPtr.h \
+        ../../../OOXML/Base/Unit.h \
+        ../../../OOXML/Base/Unit.h \
+        ../../../OOXML/Common/SimpleTypes_Drawing.h \
+        ../../../OOXML/Common/SimpleTypes_OMath.h \
+        ../../../OOXML/Common/SimpleTypes_Spreadsheet.h \
+        ../../../OOXML/Common/SimpleTypes_Vml.h \
+        ../../../OOXML/Common/SimpleTypes_Word.h \
+        ../../../OOXML/DocxFormat/App.h \
+        ../../../OOXML/DocxFormat/Comments.h \
+        ../../../OOXML/DocxFormat/Core.h \
+        ../../../OOXML/DocxFormat/Diagram/DiagramData.h \
+        ../../../OOXML/DocxFormat/Diagram/DiagramDrawing.h \
+        ../../../OOXML/DocxFormat/Document.h \
+        ../../../OOXML/DocxFormat/Docx.h \
+        ../../../OOXML/DocxFormat/DocxFlat.h \
+        ../../../OOXML/DocxFormat/Drawing/DrawingExt.h \
+        ../../../OOXML/DocxFormat/Endnote.h \
+        ../../../OOXML/DocxFormat/External/HyperLink.h \
+        ../../../OOXML/DocxFormat/FontTable.h \
+        ../../../OOXML/DocxFormat/Footnote.h \
+        ../../../OOXML/DocxFormat/HeaderFooter.h \
+        ../../../OOXML/DocxFormat/Logic/AlternateContent.h \
+        ../../../OOXML/DocxFormat/Logic/Bdo.h \
+        ../../../OOXML/DocxFormat/Logic/Dir.h \
+        ../../../OOXML/DocxFormat/Logic/FldSimple.h \
+        ../../../OOXML/DocxFormat/Logic/Hyperlink.h \
+        ../../../OOXML/DocxFormat/Logic/Paragraph.h \
+        ../../../OOXML/DocxFormat/Logic/ParagraphProperty.h \
+        ../../../OOXML/DocxFormat/Logic/Pict.h \
+        ../../../OOXML/DocxFormat/Logic/Run.h \
+        ../../../OOXML/DocxFormat/Logic/RunProperty.h \
+        ../../../OOXML/DocxFormat/Logic/Sdt.h \
+        ../../../OOXML/DocxFormat/Logic/SectionProperty.h \
+        ../../../OOXML/DocxFormat/Logic/SmartTag.h \
+        ../../../OOXML/DocxFormat/Logic/Table.h \
+        ../../../OOXML/DocxFormat/Logic/Vml.h \
+        ../../../OOXML/DocxFormat/Logic/VmlWord.h \
+        ../../../OOXML/DocxFormat/Math/OMath.h \
+        ../../../OOXML/DocxFormat/Math/oMathContent.h \
+        ../../../OOXML/DocxFormat/Math/oMathPara.h \
+        ../../../OOXML/DocxFormat/Media/ActiveX.h \
+        ../../../OOXML/DocxFormat/Numbering.h \
+        ../../../OOXML/DocxFormat/Settings/Settings.h \
+        ../../../OOXML/DocxFormat/Settings/WebSettings.h \
+        ../../../OOXML/DocxFormat/Styles.h \
+        ../../../OOXML/DocxFormat/VmlDrawing.h \
+        ../../../OOXML/PPTXFormat/DrawingConverter/ASCOfficeDrawingConverter.h \
+        ../../../OOXML/PPTXFormat/Folder.h \
+        ../../../OOXML/PPTXFormat/Logic/ClrMapOvr.h \
+        ../../../OOXML/PPTXFormat/Logic/Colors/PrstClr.h \
+        ../../../OOXML/PPTXFormat/Logic/Colors/SchemeClr.h \
+        ../../../OOXML/PPTXFormat/Logic/Colors/SrgbClr.h \
+        ../../../OOXML/PPTXFormat/Logic/CxnSp.h \
+        ../../../OOXML/PPTXFormat/Logic/Effects/AlphaModFix.h \
+        ../../../OOXML/PPTXFormat/Logic/Effects/Duotone.h \
+        ../../../OOXML/PPTXFormat/Logic/Effects/Grayscl.h \
+        ../../../OOXML/PPTXFormat/Logic/GraphicFrame.h \
+        ../../../OOXML/PPTXFormat/Logic/HeadingVariant.h \
+        ../../../OOXML/PPTXFormat/Logic/Pic.h \
+        ../../../OOXML/PPTXFormat/Logic/Shape.h \
+        ../../../OOXML/PPTXFormat/Logic/SmartArt.h \
+        ../../../OOXML/PPTXFormat/Logic/SpTree.h \
+        ../../../OOXML/PPTXFormat/Logic/SpTreeElem.h \
+        ../../../OOXML/PPTXFormat/Logic/Table/Table.h \
+        ../../../OOXML/PPTXFormat/Logic/TablePartStyle.h \
+        ../../../OOXML/PPTXFormat/Logic/TcBdr.h \
+        ../../../OOXML/PPTXFormat/Logic/TextFont.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/Anim.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/AnimClr.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/AnimEffect.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/AnimMotion.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/AnimRot.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/AnimScale.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/Audio.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/CBhvr.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/CTn.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/Par.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/Seq.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/Set.h \
+        ../../../OOXML/PPTXFormat/Logic/Timing/Timing.h \
+        ../../../OOXML/PPTXFormat/Logic/Transitions/CornerDirectionTransition.h \
+        ../../../OOXML/PPTXFormat/Logic/Transitions/EightDirectionTransition.h \
+        ../../../OOXML/PPTXFormat/Logic/Transitions/EmptyTransition.h \
+        ../../../OOXML/PPTXFormat/Logic/Transitions/OptionalBlackTransition.h \
+        ../../../OOXML/PPTXFormat/Logic/Transitions/OrientationTransition.h \
+        ../../../OOXML/PPTXFormat/Logic/Transitions/SideDirectionTransition.h \
+        ../../../OOXML/PPTXFormat/Logic/Transitions/SplitTransition.h \
+        ../../../OOXML/PPTXFormat/Logic/Transitions/Transition.h \
+        ../../../OOXML/PPTXFormat/Logic/Transitions/WheelTransition.h \
+        ../../../OOXML/PPTXFormat/Logic/Transitions/ZoomTransition.h \
+        ../../../OOXML/PPTXFormat/Logic/TxStyles.h \
+        ../../../OOXML/PPTXFormat/NotesMaster.h \
+        ../../../OOXML/PPTXFormat/NotesSlide.h \
+        ../../../OOXML/PPTXFormat/Presentation.h \
+        ../../../OOXML/PPTXFormat/Presentation/NotesSz.h \
+        ../../../OOXML/PPTXFormat/Presentation/SldSz.h \
+        ../../../OOXML/PPTXFormat/Slide.h \
+        ../../../OOXML/PPTXFormat/SlideLayout.h \
+        ../../../OOXML/PPTXFormat/SlideMaster.h \
+        ../../../OOXML/PPTXFormat/TableStyles.h \
+        ../../../OOXML/SystemUtility/File.h \
+        ../../../OOXML/SystemUtility/SystemUtility.h \
+        ../../../OOXML/XlsxFormat/CalcChain/CalcChain.h \
+        ../../../OOXML/XlsxFormat/Chart/Chart.h \
+        ../../../OOXML/XlsxFormat/Chart/ChartDrawing.h \
+        ../../../OOXML/XlsxFormat/Chart/ChartSerializeEx.h \
+        ../../../OOXML/XlsxFormat/Comments/Comments.h \
+        ../../../OOXML/XlsxFormat/Drawing/Pos.h \
+        ../../../OOXML/XlsxFormat/ExternalLinks/ExternalLinkPath.h \
+        ../../../OOXML/XlsxFormat/ExternalLinks/ExternalLinks.h \
+        ../../../OOXML/XlsxFormat/Pivot/PivotCacheDefinition.h \
+        ../../../OOXML/XlsxFormat/Pivot/PivotCacheRecords.h \
+        ../../../OOXML/XlsxFormat/Pivot/PivotTable.h \
+        ../../../OOXML/XlsxFormat/SharedStrings/SharedStrings.h \
+        ../../../OOXML/XlsxFormat/Styles/Borders.h \
+        ../../../OOXML/XlsxFormat/Styles/CellStyles.h \
+        ../../../OOXML/XlsxFormat/Styles/Colors.h \
+        ../../../OOXML/XlsxFormat/Styles/Fills.h \
+        ../../../OOXML/XlsxFormat/Styles/Fonts.h \
+        ../../../OOXML/XlsxFormat/Styles/NumFmts.h \
+        ../../../OOXML/XlsxFormat/Styles/Styles.h \
+        ../../../OOXML/XlsxFormat/Styles/Xfs.h \
+        ../../../OOXML/XlsxFormat/Styles/dxf.h \
+        ../../../OOXML/XlsxFormat/Workbook/Workbook.h \
+        ../../../OOXML/XlsxFormat/Worksheets/Sparkline.h \
+        ../../../OOXML/XlsxFormat/Worksheets/Worksheet.h \
+        ../../../OOXML/XlsxFormat/Xlsx.h \
+        ../../../OOXML/XlsxFormat/XlsxFlat.h
