--- a/X2tConverter/src/main.cpp	2026-05-30 17:28:43.778328619 +0800
+++ b/X2tConverter/src/main.cpp	2026-05-29 12:08:52.579070615 +0800
@@ -30,7 +30,7 @@
  *
  */
 
-#include "../../Common/3dParty/misc/proclimits.h"
+// #include "../../Common/3dParty/misc/proclimits.h" CryptPad: Disabled to fix build
 #include "../../DesktopEditor/common/StringExt.h"
 #include "../../DesktopEditor/common/SystemUtils.h"
 #include "ASCConverters.h"
@@ -99,11 +99,11 @@
 	if (sMemoryLimit.empty())
 		sMemoryLimit = NSSystemUtils::gc_EnvMemoryLimitDefault;
 
-#if !defined(_DEBUG) && !defined(__ANDROID__) && !defined(_IOS)
-	long long nMemoryLimit;
-	if (NSStringExt::FromHumanReadableByteCount(sMemoryLimit, nMemoryLimit) && nMemoryLimit > 0)
-		limit_memory((size_t)nMemoryLimit);
-#endif
+// #if !defined(_DEBUG) && !defined(__ANDROID__) && !defined(_IOS) CryptPad: disabled for WASM build
+// 	long long nMemoryLimit;
+// 	if (NSStringExt::FromHumanReadableByteCount(sMemoryLimit, nMemoryLimit) && nMemoryLimit > 0)
+// 		limit_memory((size_t)nMemoryLimit);
+// #endif
 	std::wstring sArg1, sArg2;
 
 #if !defined(_WIN32) && !defined(_WIN64)
