--- a/MsBinaryFile/Common/Vml/PPTShape/BinaryReader.cpp	2026-05-30 17:30:59.384909066 +0800
+++ b/MsBinaryFile/Common/Vml/PPTShape/BinaryReader.cpp	2026-05-29 12:08:50.301047229 +0800
@@ -35,61 +35,5 @@
 
 namespace ODRAW
 {
-	CBinaryReader::CBinaryReader(BYTE* pBuffer, DWORD lCount)
-	{
-		m_pBuffer = pBuffer;
-		m_lCount = lCount;
-
-		m_lOrigin = 0;
-	}
-	CBinaryReader::~CBinaryReader()
-	{
-		m_pBuffer = NULL;
-		m_lCount = 0;
-	}
-	LONG CBinaryReader::ReadLONG() // int32 подразумевается
-	{
-		DWORD lOldOrigin = m_lOrigin;
-		m_lOrigin += 4;
-
-		BINARY_READER_CHECK_OUT_RANGE(m_lOrigin, m_lCount)
-
-		return *(_INT32*)(m_pBuffer + lOldOrigin);
-	}
-	DWORD CBinaryReader::ReadDWORD()
-	{
-		DWORD lOldOrigin = m_lOrigin;
-		m_lOrigin += 4;
-
-		BINARY_READER_CHECK_OUT_RANGE(m_lOrigin, m_lCount)
-
-		return *(DWORD*)(m_pBuffer + lOldOrigin);
-	}
-	WORD CBinaryReader::ReadWORD()
-	{
-		DWORD lOldOrigin = m_lOrigin;
-		m_lOrigin += 2;
-
-		BINARY_READER_CHECK_OUT_RANGE(m_lOrigin, m_lCount)
-
-		return *(WORD*)(m_pBuffer + lOldOrigin);
-	}
-	SHORT CBinaryReader::ReadSHORT()
-	{
-		DWORD lOldOrigin = m_lOrigin;
-		m_lOrigin += 2;
-
-		BINARY_READER_CHECK_OUT_RANGE(m_lOrigin, m_lCount)
-
-		return *(short*)(m_pBuffer + lOldOrigin);
-	}
-	double CBinaryReader::ReadDOUBLE()
-	{
-		DWORD lOldOrigin = m_lOrigin;
-		m_lOrigin += sizeof(double);
-
-		BINARY_READER_CHECK_OUT_RANGE(m_lOrigin, m_lCount)
-
-		return *(SHORT*)(m_pBuffer + lOldOrigin);
-	}
+    // CryptPad: Remove content here because it is already defined in BinaryReader.h and causes errors here
 }
