--- a/X2tConverter/src/lib/html.h	2026-05-30 17:31:01.482094168 +0800
+++ b/X2tConverter/src/lib/html.h	2026-05-29 12:08:52.574812628 +0800
@@ -33,7 +33,7 @@
 
 #include "../../../Common/3dParty/pole/pole.h"
 #include "../../../EpubFile/CEpubFile.h"
-#include "../../../Fb2File/Fb2File.h"
+// #include "../../../Fb2File/Fb2File.h"
 #include "../../../HtmlFile2/htmlfile2.h"
 #include "../../../Common/3dParty/md/md2html.h"
 #include "common.h"
@@ -147,13 +147,15 @@
 	}
 	_UINT32 fb2docx_dir(const std::wstring& sFrom, const std::wstring& sTo, InputParams& params, ConvertParams& convertParams)
 	{
-		CFb2File fb2File;
-		fb2File.SetTmpDirectory(convertParams.m_sTempDir);
-		return S_OK == fb2File.Open(sFrom, sTo) ? S_OK : AVS_FILEUTILS_ERROR_CONVERT;
+		// CFb2File fb2File;
+		// fb2File.SetTmpDirectory(convertParams.m_sTempDir);
+		// return S_OK == fb2File.Open(sFrom, sTo) ? S_OK : AVS_FILEUTILS_ERROR_CONVERT;
+		return 0;
 	}
 	_UINT32 fb2docx(const std::wstring& sFrom, const std::wstring& sTo, InputParams& params, ConvertParams& convertParams)
 	{
-		return NSCommon::format2ooxml(sFrom, sTo, params, convertParams, L"docx", fb2docx_dir);
+		// return NSCommon::format2ooxml(sFrom, sTo, params, convertParams, L"docx", fb2docx_dir);
+		return 0;
 	}
 
 	// doct_bin => html
@@ -176,7 +178,7 @@
 
 		std::wstring sResult;
 		oDoctRenderer.Execute(sXml, sResult);
-		
+
 		if (sResult.find(L"error") != std::wstring::npos)
 		{
 			std::wcerr << L"DoctRenderer:" << sResult << std::endl;
@@ -217,18 +219,19 @@
 	// doct_bin -> fb2
 	_UINT32 doct_bin2fb(const std::wstring& sFrom, const std::wstring& sTo, InputParams& params, ConvertParams& convertParams)
 	{
-		_UINT32 nRes = doct_bin2html_internal(sFrom, sTo, params, convertParams);
-		if (0 != nRes)
-			return nRes;
+		// _UINT32 nRes = doct_bin2html_internal(sFrom, sTo, params, convertParams);
+		// if (0 != nRes)
+		// 	return nRes;
+
+		// std::wstring sHtmlFile = combinePath(convertParams.m_sTempDir, L"index.html");
+
+		// CFb2File fb2File;
+		// fb2File.SetTmpDirectory(convertParams.m_sTempDir);
+		// if (S_FALSE == fb2File.FromHtml(sHtmlFile, sTo, params.m_sTitle ? *params.m_sTitle : L""))
+		// 	nRes = AVS_FILEUTILS_ERROR_CONVERT;
 
-		std::wstring sHtmlFile = combinePath(convertParams.m_sTempDir, L"index.html");
-
-		CFb2File fb2File;
-		fb2File.SetTmpDirectory(convertParams.m_sTempDir);
-		if (S_FALSE == fb2File.FromHtml(sHtmlFile, sTo, params.m_sTitle ? *params.m_sTitle : L""))
-			nRes = AVS_FILEUTILS_ERROR_CONVERT;
-
-		return nRes;
+		// return nRes;
+		return 0;
 	}
 	// doct_bin -> html
 	_UINT32 doct_bin2html(const std::wstring& sFrom, const std::wstring& sTo, InputParams& params, ConvertParams& convertParams)
